src/main/io/displayport_msp_bf_compat.c

```diff
@@ -90,13 +90,13 @@ uint8_t getBfCharacter(uint8_t ch, uint8_t page)
 
     case SYM_SNR:
         return BF_SYM_SNR;
-
+*/
     case SYM_AH_DECORATION_UP:
-        return BF_SYM_AH_DECORATION_UP;
+        return BF_SYM_ARROW_SMALL_UP;
 
     case SYM_AH_DECORATION_DOWN:
-        return BF_SYM_AH_DECORATION_DOWN;
-*/
+        return BF_SYM_ARROW_SMALL_DOWN;
+
```

Comment:
This should be a separate PR. As it is not in the original scope.

Reply:
Got it, thank you, will do now. 👍